### ISSUE TITLE
Composer require gives errors. 

### ISSUE THREAD

**Issue by MrBean14** [barryvdh/laravel-dompdf#402]
Hi all,

I'm quite new with laravel and wrote my first app. 
As I used FDPF for many years in the normal PHP way, I wanted to import it in this laravel app too. 

I'm using Laravel 5.4.33 with PHP 7.0 but when I run the "composer require barryvdh/laravel-dompdf" command, I get following issues. I have followed many "possible solutions" but still it is not working.

Oh, BTW, I'm using Scotch.box ...

This is the error 👍 

`vagrant@scotchbox:/var/www/public/plevoets$ composer require barryvdh/laravel-dompdf
Using version ^0.8.1 for barryvdh/laravel-dompdf
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - phpunit/php-code-coverage 4.0.8 requires ext-dom * -> the requested PHP extension dom is missing from your system.
    - phpunit/php-code-coverage 4.0.8 requires ext-dom * -> the requested PHP extension dom is missing from your system.
    - phpunit/php-code-coverage 4.0.8 requires ext-dom * -> the requested PHP extension dom is missing from your system.
    - Installation request for phpunit/php-code-coverage (locked at 4.0.8) -> satisfiable by phpunit/php-code-coverage[4.0.8].

  To enable extensions, verify that they are enabled in your .ini files:
    - /etc/php/7.0/cli/php.ini
    - /etc/php/7.0/cli/conf.d/10-mysqlnd.ini
    - /etc/php/7.0/cli/conf.d/10-opcache.ini
    - /etc/php/7.0/cli/conf.d/10-pdo.ini
    - /etc/php/7.0/cli/conf.d/20-bcmath.ini
    - /etc/php/7.0/cli/conf.d/20-bz2.ini
    - /etc/php/7.0/cli/conf.d/20-calendar.ini
    - /etc/php/7.0/cli/conf.d/20-ctype.ini
    - /etc/php/7.0/cli/conf.d/20-curl.ini
    - /etc/php/7.0/cli/conf.d/20-enchant.ini
    - /etc/php/7.0/cli/conf.d/20-exif.ini
    - /etc/php/7.0/cli/conf.d/20-fileinfo.ini
    - /etc/php/7.0/cli/conf.d/20-ftp.ini
    - /etc/php/7.0/cli/conf.d/20-gd.ini
    - /etc/php/7.0/cli/conf.d/20-gettext.ini
    - /etc/php/7.0/cli/conf.d/20-iconv.ini
    - /etc/php/7.0/cli/conf.d/20-igbinary.ini
    - /etc/php/7.0/cli/conf.d/20-imagick.ini
    - /etc/php/7.0/cli/conf.d/20-imap.ini
    - /etc/php/7.0/cli/conf.d/20-intl.ini
    - /etc/php/7.0/cli/conf.d/20-json.ini
    - /etc/php/7.0/cli/conf.d/20-ldap.ini
    - /etc/php/7.0/cli/conf.d/20-mbstring.ini
    - /etc/php/7.0/cli/conf.d/20-mcrypt.ini
    - /etc/php/7.0/cli/conf.d/20-mongodb.ini
    - /etc/php/7.0/cli/conf.d/20-msgpack.ini
    - /etc/php/7.0/cli/conf.d/20-mysqli.ini
    - /etc/php/7.0/cli/conf.d/20-odbc.ini
    - /etc/php/7.0/cli/conf.d/20-pdo_mysql.ini
    - /etc/php/7.0/cli/conf.d/20-pdo_odbc.ini
    - /etc/php/7.0/cli/conf.d/20-pdo_pgsql.ini
    - /etc/php/7.0/cli/conf.d/20-pdo_sqlite.ini
    - /etc/php/7.0/cli/conf.d/20-pgsql.ini
    - /etc/php/7.0/cli/conf.d/20-phar.ini
    - /etc/php/7.0/cli/conf.d/20-posix.ini
    - /etc/php/7.0/cli/conf.d/20-pspell.ini
    - /etc/php/7.0/cli/conf.d/20-readline.ini
    - /etc/php/7.0/cli/conf.d/20-redis.ini
    - /etc/php/7.0/cli/conf.d/20-shmop.ini
    - /etc/php/7.0/cli/conf.d/20-sockets.ini
    - /etc/php/7.0/cli/conf.d/20-sqlite3.ini
    - /etc/php/7.0/cli/conf.d/20-sysvmsg.ini
    - /etc/php/7.0/cli/conf.d/20-sysvsem.ini
    - /etc/php/7.0/cli/conf.d/20-sysvshm.ini
    - /etc/php/7.0/cli/conf.d/20-tidy.ini
    - /etc/php/7.0/cli/conf.d/20-tokenizer.ini
    - /etc/php/7.0/cli/conf.d/20-xmlrpc.ini
    - /etc/php/7.0/cli/conf.d/20-zip.ini
    - /etc/php/7.0/cli/conf.d/25-memcached.ini
  You can also run `php --ini` inside terminal to see which files are used by PHP in CLI mode.

Installation failed, reverting ./composer.json to its original content.`

**Comment by danielsdeboer** [comment]
Looks like you need to enable the `ext-dom` PHP extension. Which may mean simply uncommenting that line in your php configuration file, or installing the extension. I've not used Scotch.box but here is what Google found:

https://github.com/scotch-io/scotch-box/issues/334

**Comment by MrBean14** [comment]
Hi, thanks for your answer. 
Just after my request I found a document describing to install php7.0-xml. 
The error above was then solved and I was able to run the "composer require". 

Then, in my config.app.php, I added : 

```
Barryvdh\DomPDF\ServiceProvider::class,
'PDF' => Barryvdh\DomPDF\Facade::class,
```

Then I tried to put following code in my Controller to do a quick test : 

```
use Barryvdh\DomPDF\PDF;
...
$facturen = Factuur::all();
$pdf = PDF::loadView('pdf.facturen', $facturen);
return $pdf->download('invoice.pdf');
```

Unfortunately I now get following error when opening the view 👍 

_(1/1) FatalThrowableErrorClass 'DOMDocument' not found
--
in Dompdf.php (line 488)
at Dompdf->loadHtml('<meta http-equiv="Content-Type" content="text/html;charset=UTF-8">', 'UTF-8')in PDF.php (line 96)
at PDF->loadHTML('', null)in PDF.php (line 124)
at PDF->loadView('pdf.facturen', object(Collection))in Facade.php (line 31)_

I started looking around and somebody adviced me to try the code : 

_dd(class_exists('DOMDocument'));_

This results to false.

I do not find the way to solve it. 
In my php.ini I can not find back this extension.

Best regards,

Davy
